data/mods/My_Sweet_Cataclysm/sweet_monsters.json

```diff
@@ -414,5 +414,22 @@
     "anger_triggers": [ "STALK", "PLAYER_WEAK", "HURT", "PLAYER_CLOSE" ],
     "death_function": [ "NORMAL" ],
     "flags": [ "SEES", "SMELLS", "HEARS", "CLIMBS", "PATH_AVOID_FIRE", "PATH_AVOID_FALL" ]
+  },
+  {
+    "id": "mon_cow_calf_choc",
```

Comment:
You need to add an `"harvest"` entry to both monsters or they'll butcher into meat and bones instead of chocolate.

Reply:
I was envisioning a regular cow that happened to produce chocolate milk instead of regular milk, so this was intended. If you would strongly prefer for them to be made of chocolate to fit into the mod better, lmk and I'll do it.

Reply:
Yes please make the chocolate cow be made of chocolate. You also need a candy item to spawn them, I suggest adding cow shaped chocolate to vanilla and use that.

Reply:
The current iteration has them spawning from chocolate organs, which I prefer. I will switch it to chocolate cow candies if you prefer.

Reply:
Chocolate organs are a good idea, but that doesn't quite make sense that they spawn whole cows again. So here's a proposition calves spawn from chocolate cow candies and evolve into cows, cows can spawn more claves like regular cows and butcher into chocolate organs that don't spawn anything they just rot and disapear. And since those candy creatures have organs and stuff instead of just being pure candy they should appear much later, as the sweet cataclysm progresses.

Reply:
My logic for that was that the chocolate organs were the parts of the cow that still had life, and therefore were the parts that could still reorganize into a new calf. Basing their spawning primarily off of candies like the rest of the MSC candies was a lot easier to set up than setting up the rest of the chocolate organs though, so they now spawn from candies, and no longer spawn with the cow monster group. I set the spawn rate of their candy lower than the others in most cases since chocolate cows aren't hostile and are kind of just free resources.